vendor/github.com/docker/go-connections/nat/nat.go

```diff
@@ -15,6 +15,21 @@ type PortBinding struct {
 	HostIP string `json:"HostIp"`
 	// HostPort is the host port number
 	HostPort string
+
+	// Mapper is an optional field that defines which portmapper should be used
```

Comment:
can we avoid adding things to go-connections somehow? We're trying to get rid of it